python/fusion_engine_client/messages/configuration.py

```diff
@@ -873,6 +900,27 @@ class RateResponseEntry(NamedTuple):
     configured_rate: MessageRate = MessageRate.OFF
     effective_rate: MessageRate = MessageRate.OFF
 
+    __parent_str__ = object.__str__
+
+    def __str__(self):
+        enum = None
```

Comment:
Since this logic is repeated it probably makes sense to make a helper function.

Reply:
Makes sense, added a `_get_message_type_string()` function.